tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -0,0 +1,95 @@
+| **linux**                     | Linux operating system                                                                               |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **android**                   | Android operating system                                                                             |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **ios**                       | iOS operating system                                                                                 |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **web**                       | Web browser                                                                                          |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+
+Debugging symbols
+^^^^^^^^^^^^^^^^^
+
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| Flag                          | Description                                                                                          |
++===============================+======================================================================================================+
+| **debug**                     | Build with debug symbols                                                                             |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **release**                   | Optimized build without debug symbols                                                                |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+
+Architecture
+^^^^^^^^^^^^
+
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| Flag                          | Description                                                                                          |
++===============================+======================================================================================================+
+| **x86_64**                    | 64-bit Windows                                                                                       |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **x86_32**                    | 32-bit Windows                                                                                       |
```

Comment:
The flags are architecture-specific, not Windows-specific. x86_64 means 64-bit x86, not 64-bit Windows.

```suggestion
| Flag                          | Description                                                                                          |
+===============================+======================================================================================================+
| **x86_64**                    | 64-bit x86                                                                                           |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **x86_32**                    | 32-bit x86                                                                                           |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **arm32**                     | 32-bit Arm                                                                                           |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **arm64**                     | 64-bit Arm                                                                                           |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **rv64**                      | 64-bit RISC-V                                                                                        |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **ppc32**                     | 32-bit PowerPC                                                                                       |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **ppc64**                     | 64-bit PowerPC                                                                                       |
+-------------------------------+------------------------------------------------------------------------------------------------------+
| **wasm32**                    | 32-bit WebAssembly                                                                                   |
```

Reply:
Noticed already. Thank you!